### ISSUE TITLE
[CIT] Enchantment Glints and Custom Animations not working

### ISSUE THREAD

**Issue by rata536** [sp614x/optifine#7327]
Setting a custom enchantment glint via CIT does not work, showing the standard glint texture instead.

![image](https://user-images.githubusercontent.com/59150478/227626709-97bedcdc-28e0-4a4e-b9c2-3a027921be1b.png)

![image](https://user-images.githubusercontent.com/59150478/227626775-88c54fe6-b96e-464f-9071-cff393a7b788.png)

![image](https://user-images.githubusercontent.com/59150478/227626882-71b6db3d-4f2f-4831-9eee-1251e38ce43d.png)
This axe has Mending which should trigger the custom glint texture

Second issue: custom animation is pasting the animation on the loading screen rather than the supposed target texture:
![image](https://user-images.githubusercontent.com/59150478/227627133-94b28dc2-bb14-4f0e-bfc0-7a20a1526ccb.png)
![image](https://user-images.githubusercontent.com/59150478/227627158-7c449b1e-1412-4a30-af82-9028f0758382.png)
![image](https://user-images.githubusercontent.com/59150478/227627181-ae255768-3d3b-4354-9c2b-3539946fd4bc.png)

These behavious have been like this since 1.16.

**Comment by IranMine123** [comment]
Hello, for the Enchantments Glint, as I know, they removed this feature on version 1.14. there is an issue report for that (#6835).

**Comment by toutlamer** [comment]
I know there was changes to the enchantments thingy in 1.19.4 (https://minecraft.fandom.com/wiki/Pack_format)

**Comment by rata536** [comment]
> 

No, the feature was working on 1.14 and I *think* in 1.15 for the last time. Also, the feature is still listed in the docs.
